### PR TITLE
Added Set Badge Number

### PR DESCRIPTION
**Pull request by bocodigital**
I added set badge number to this from the Firebase Plugin, thought you might want it.
Thank

### CODE REVIEW

File: src/ios/AppDelegate+FCMPlugin.m [hunk 1]
```diff
@@ -46,7 +46,7 @@ - (BOOL)application:(UIApplication *)application customDidFinishLaunchingWithOpt
     }
 
     // [START configure_firebase]
-    [FIRApp configure];
+    //[FIRApp configure];
```
Comment (grrrian) [inline src/ios/AppDelegate+FCMPlugin.m]: why are you removing this line?

Comment (bocodigital) [reply src/ios/AppDelegate+FCMPlugin.m]: Sorry, I was using this in conjunction with the firebase plugin before so the app crashes when they both try and configure. I can undo and resubmit the pull or however you'd like to handle it.

Comment (grrrian) [reply src/ios/AppDelegate+FCMPlugin.m]: Are you setting the badge to anything other than 0 anyway? I'm thinking of writing my own PR that just resets the badge on app open.

**Comment by mrgcohen** [comment]
I wolud love this!

**Pull request closed without merging**